readme.md

```diff
@@ -104,6 +104,13 @@ Or, you can simply mount `/plots` path to your host machine.
 
 Set the environment variable `recursive_plot_scan` to `true` to enable the recursive plot scan configuration option.
 
+### Adding mounts while running
+
+By default, Docker requires a container restart to discover newly mounted filesystems under a configured bind-mount. Setting the bind-propagation option to `rslave` enables dynamic addition of sub-mounts while the container is running (Linux systems only).
```

Comment:
```suggestion
By default, Docker requires a container restart to discover newly mounted filesystems under a configured bind-mount. Setting the bind-propagation option to `rslave` enables dynamic addition of sub-mounts while the container is running (Linux systems only). [See Docker Bind Mounts documentation for more information.](https://docs.docker.com/storage/bind-mounts/#configure-bind-propagation)
```
In my opinion, we probably don't need to document every feature of vanilla Docker but I could see this in particular being useful for farmers. But I tend to prefer linking to the documentation.